### PR TITLE
Challenge

### PR DESCRIPTION
**Pull request by anessao**
@zoeames Finished quiz

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+							<a href="http://www.spacex.com/"><span>SpaceX</span></a>
+						</li>
+					</ul>
+				</section>
+			</article>
+		</div>
+		<footer class="main-footer">
+			<section class="contact">
+				<h5>Contact:</h5>
+				<a href= "mailto:shadowglacierdesigns@gmail.com">Send Email</a>
```
Comment (zoeames) [inline index.html]: this looks kinda awkward.  How about a letter icon that when you click on it does the mailto?  That would better mirror the icons you have in the  connect section

**Comment by zoeames** [comment]
VERY NICE SITE!   Just on comment on the html.  I really like how you have commented your CSS file so its clear what sections go to what.  How about doing the same thing in your HTML file

**Comment by anessao** [comment]
@zoeames Thank you for reviewing my quiz!  I've updated the html with section comments and included a new email link.  Let me know if there's anything else I can/should do!

**Comment by zoeames** [comment]
Very nice!  I am approving and merging for you. 👍

**Pull request merged**